frontend/components/panels/AllowanceAmountCheckPanel.stories.js

```diff
@@ -0,0 +1,119 @@
+import { storiesOf } from '@storybook/vue';
+import faker from 'faker';
+import BigNumber from 'bignumber.js';
+import AllowanceAmountCheckPanel from './AllowanceAmountCheckPanel';
+
+const common = {
+    components: { AllowanceAmountCheckPanel },
+    data() {
+        return {
+            allowanceAmount: new BigNumber(faker.finance.amount(0, 100)),
+            desiredAmount: new BigNumber(faker.finance.amount(100, 200)),
+            isLoading: false,
+        };
+    },
+    methods: {
+        setAllowanceAmount(amount) {
+            this.isLoading = true;
+            setTimeout(() => {
+                this.allowanceAmount = amount;
+                this.isLoading = false;
+            }, 1000);
+        },
+    },
+};
+
+storiesOf('Panels/AllowanceAmountCheckPanel', module)
+    .add('Within Allowance', () => ({
+        ...common,
+        data() {
+            return {
+                allowanceAmount: new BigNumber(faker.finance.amount(100, 200)),
+                desiredAmount: new BigNumber(faker.finance.amount(0, 100)),
+                isLoading: false,
+            };
+        },
+        template: `
+        <AllowanceAmountCheckPanel 
+            :allowance-amount="allowanceAmount" 
+            :desired-amount="desiredAmount"
+            :is-loading="isLoading"
+            @setAllowanceAmount="setAllowanceAmount" 
+        />`,
```

Comment:
If template stays unchanged over the stories, you can move it to the `common` object above and define only once